src/wallet/rpcwallet.cpp

```diff
@@ -305,6 +305,199 @@ UniValue sendtoaddress(const UniValue& params, bool fHelp)
     return wtx.GetHash().GetHex();
 }
 
+UniValue listaddresses(const UniValue& params, bool fHelp)
+{
+    if (!EnsureWalletIsAvailable(fHelp))
+        return NullUniValue;
+
+    if (fHelp)
+        throw runtime_error(
+            "listaddresses\n"
+            "\nLists the addresses managed by this wallet by source, including \n"
+            "those generated from randomness by this wallet, Sapling addresses \n"
+            "generated from the legacy HD seed, imported watchonly transparent \n"
+            "addresses, shielded addresses tracked using imported viewing keys, \n"
+            "and addresses derived from the wallet's mnemonic seed for releases \n"
+            "version 5.0.0 and above. \n"
```

Comment:
This will actually end up being 4.5.2, since the wallet will start using the mnemonic as soon as it is available (it isn't gated on network type).

Reply:
```suggestion
            "version 4.5.2 and above. \n"
```